azure/README.md

```diff
@@ -102,7 +104,7 @@ For more details, visit [Microsoft Azure - dask cloud provider](https://cloudpro
 jupyter notebook --ip 0.0.0.0 --port 8888 --no-browser --allow-root --NotebookApp.token=''
 ```
 
-6. Navigate to the notebook `azure_mnmg.ipynb` under `azure/notebooks`
+6. Navigate either to either of the two notebooks `Azure-MNMG-RF.ipynb` or `azure_mnmg_xgboost.ipynb` under `./notebooks`.
```

Comment:
Minor change - Can you rename the notebook to Azure-MNMG-XGBoost for consistency?

Reply:
Done.